### DISCUSSION TITLE
sort by name inside of albums?

### DISCUSSION THREAD

**Discussion by lonely-ux** [ente-io/ente#1880] [Enhancements]
Like in the title. I have some old photos that have lost their date taken metadata but the date they were taken is in the file name. So for other gallery apps that's what I've used. This option is available for albums so it makes sense that it's used inside of albums as well?

Upvoted a previous discussion regarding this but that was 6 months ago with zero talk so reviving it. 

Thanks!

**Comment by dedelstein** [comment]
+1

**Comment by ua741** [comment]
Dup: https://github.com/ente-io/ente/discussions/638
